trunk/NDHMS/Routing/module_channel_routing.F

```diff
@@ -266,7 +266,7 @@ subroutine SUBMUSKINGCUNGE(    &
 
            Twl = Bw + 2.0*z*h_0      !--top surface water width of the channel inflow
 
-            if(h_0 .gt. bfd) then !water outside of defined channel
+            if ( (h_0 .gt. bfd) .and. (TwCC .gt. 0.0) .and. (nCC .gt. 0.0) ) then !water outside of defined channel
```

Comment:
@aubreyd - just so I understand this correctly. If there is a floodplain width of zero, then we just extend the vertical extent of the trapezoidal main channel? So, even if the depth of water is greater than bank full depth, we still use the area and wetted perimeter calculations for the trapezoidal main channel (with h_0 > bfd)?